tree/30_generic_methods/service_check_started_at_boot.cf

```diff
@@ -38,7 +38,10 @@ bundle agent service_check_started_at_boot(service_name)
     pass1.!systemctl_utility_present.!chkconfig_utility_present.lsitab_utility_present::
       "command_to_check"   string => "/usr/sbin/lsitab -a | egrep '^${service_name}:[0-9]+:(respawn|boot|bootwait|wait|once|initdefault|sysinit):'";
 
-    pass1.(!(systemctl_utility_present|chkconfig_utility_present|lsitab_utility_present)|broken_systemctl)::
+    pass1.!systemctl_utility_present.!chkconfig_utility_present.!lsitab_utility_present.svcs_utility_present::
+      "command_to_check"   string => "${paths.path[svcs]} -l ${service_name} | egrep '^enabled [ 	]+(true|false \(temporary\))'";
+
+    pass1.(!(systemctl_utility_present|chkconfig_utility_present|lsitab_utility_present|svcs_utility_present)|broken_systemctl)::
```

Comment:
Could you rewrite this to use the same convention as above (!x.!y instead of !(x|y) )